src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psd1

```diff
@@ -48,10 +48,18 @@ FormatsToProcess = @()
 NestedModules = @('Microsoft.Azure.Functions.PowerShellWorker.psm1', 'Microsoft.Azure.Functions.PowerShellWorker.dll')
 
 # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
-FunctionsToExport = @('Start-NewOrchestration', 'New-OrchestrationCheckStatusResponse')
+FunctionsToExport = @(
+    'Start-NewOrchestration',
+    'New-OrchestrationCheckStatusResponse')
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = @('Push-OutputBinding', 'Get-OutputBinding', 'Trace-PipelineObject', 'Set-FunctionInvocationContext', 'Invoke-ActivityFunction')
+CmdletsToExport = @(
+    'Push-OutputBinding',
+    'Get-OutputBinding',
+    'Trace-PipelineObject',
+    'Set-FunctionInvocationContext',
+    'Invoke-ActivityFunction',
+    'Wait-ActivityFunction')
```

Comment:
Since we are adding the `Wait-ActivityFunction` cmdlet, we should probably increase the module version.

Reply:
Done.